lib/oneview-sdk/resource/api200/logical_switch.rb

```diff
@@ -49,7 +49,7 @@ def create
       # Updates this object using the data that exists on OneView
       # @note Will overwrite any data that differs from OneView
       # @return [Resource] self
-      def refresh
+      def refresh_data_state!
```

Comment:
I would prefer calling it `refresh_state` to follow the pattern with the other refreshes